README.md

```diff
@@ -10,11 +10,25 @@
 
 [![Cerner OSS](https://badgen.net/badge/Cerner/OSS/blue)](http://engineering.cerner.com/2014/01/cerner-and-open-source/)
 [![License](https://badgen.net/github/license/cerner/terra-graphs)](https://github.com/cerner/terra-graphs/blob/main/LICENSE)
-[![Build Status](https://badgen.net/travis/cerner/terra-graphs)](https://travis-ci.com/cerner/terra-graphs)
-[![devDependencies status](https://badgen.net/david/dev/cerner/terra-graphs)](https://david-dm.org/cerner/terra-graphs?type=dev)
+[![Build Status](https://travis-ci.com/cerner/terra-graphs.svg?branch=main)](https://travis-ci.com/cerner/terra-graphs)
 [![lerna](https://badgen.net/badge/maintained%20with/lerna/cc00ff)](https://lerna.js.org/)
+[![npm latest version](https://img.shields.io/npm/v/@cerner/carbon-graphs/latest.svg)](https://www.npmjs.com/package/@cerner/carbon-graphs)
```

Comment:
this is mixing metaphors here, since this is a mono repo housing multiple packages it's not accurate to show only the carbon-graphs package version in the read me.

Reply:
This readme instead point to the carbon graphs readme.

Reply:
See the terra-core readme: https://github.com/cerner/terra-core/blob/main/README.md

Reply:
I agree we should display the terra-package version here and

We should add carbon-package like this: https://github.com/cerner/terra-core/blob/main/README.md
![image](https://user-images.githubusercontent.com/62761486/97496896-ab229c80-1937-11eb-85b8-95f5339d7f74.png)

Reply:
Updated it here: https://github.com/cerner/terra-graphs/pull/5/commits/121c4231f3a5ad87feedb79b2d791aee701af219